cli/tests/pcluster/config/test_config_patch.py

```diff
@@ -290,93 +356,79 @@ def _test_different_labels(base_conf, target_conf):
         _test_more_target_sections,
         _test_incompatible_ebs_sections,
         _test_equal_configs,
-        _test_different_labels,
+        _test_different_names,
     ],
 )
-def test_adaptation(mocker, test_datadir, pcluster_config_reader, test):
-    _do_mocking_for_tests(mocker)
-    base_config_file_name = "pcluster.config.base.ini"
-    duplicate_config_file(base_config_file_name, test_datadir)
-    target_config_file_name = "pcluster.config.dst.ini"
-    duplicate_config_file(target_config_file_name, test_datadir)
+def test_adaptation(test_datadir, pcluster_config_reader, test):
```

Comment:
Is this test still needed? I got the impression that the adaptation phase has been removed.

Reply:
test_adaptation is a parametrized tests that executes:
    [
        _test_less_target_sections,
        _test_more_target_sections,
        _test_incompatible_ebs_sections,
        _test_equal_configs,
        _test_different_names,
    ],